### DISCUSSION TITLE
simple sftp or web-daw?

### DISCUSSION THREAD

**Discussion by ghost** [BlueSCSI/BlueSCSI-v2#203] [Ideas]
Hi,

I have been using these sweed units for a while now (in my se/30, powerbook 170, Yamaha A3000 sampler, externally coupled to my Roland XV5080) and i love the functionality and ease of use. Thank you for this awersome device :).

I have a mixed bag of them, a few without wifi capability and some with it. So i was thinking would it be even remotely possible to create a small sftp-server for the wifi ones, or even some simple web-ui, enabling them to accept new images and assign them accordingly this way?

For the wifi-cabable units, this would allow them to (i presume) swap images without having to extract the BlueScsi board out of its casing (which may be more or less cumbersome depending...) and NOT having to resort to having tf/sf extender cables flopping arond (sometimes these brings issues also).

**Comment by erichelgeson** [comment]
The wifi connection of the Pico is 802.11b - which would be painfully slow to transfer images. We will likely have a remote management over wifi (switching cd's, etc) but (likely) not file transfers. 

A nice case with a front accessible SD card or an extender is likely your best bet. Also you can hot swap the SD card (when no disk access) if that helps your use case.

**Comment by ghost** [comment]
Hi,

i understand and i makes perfect sense also.

Even if it would be possible in the future to assign "active" images from a pool of images available on the card (given of course that nowadays there can be lot of available storage space on SD-cards in various flavours) that would be a huge benefit (if such a remote managment UI would appear). 

I'll keep my eyes peeled here meanwhile :)

Thx

br

Daniel
